core/src/main/scala/io/odin/loggers/AsyncLogger.scala

```diff
@@ -13,39 +13,44 @@ import scala.concurrent.duration._
   *
   * Use `AsyncLogger.withAsync` to instantiate it safely
   */
-case class AsyncLogger[F[_]](queue: ConcurrentQueue[F, LoggerMessage], timeWindow: FiniteDuration, inner: Logger[F])(
-    implicit F: Concurrent[F],
-    timer: Timer[F],
-    contextShift: ContextShift[F]
+case class AsyncLogger[F[_]](queue: Queue[F, LoggerMessage], timeWindow: FiniteDuration, inner: Logger[F])(
+    implicit F: Async[F]
```

Comment:
Not sure if you need the full Async here

Reply:
The `runF` method uses `F.start` under the hood. If we move the event consumer loop outside of the class, the constraints can be relaxed to `Monad` and `Clock`. 
Related comment: https://github.com/valskalla/odin/pull/273#discussion_r607014838

Reply:
`start` is from `Spawn`, `Async` is way more powerful than that ;)